app/cdap/components/DataPrep/DataPrepSidePanel/SamplingTab/SamplingTab.scss

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
```

Comment:
Copyright date for new code should be 2022.

Reply:
Good catch -- fixed.